src/runtime_src/core/edge/common/aie_parser.cpp

```diff
@@ -446,6 +481,19 @@ get_tiles(const xrt_core::device* device, const std::string& graph_name)
   return ::get_tiles(aie_meta, graph_name);
 }
 
+std::vector<tile_type>
+get_event_tiles(const xrt_core::device* device, const std::string& graph_name,
+                module_type type)
+{
+  auto data = device->get_axlf_section(AIE_METADATA);
+  if (!data.first || !data.second)
+    return std::vector<tile_type>();
+
+  pt::ptree aie_meta;
+  read_aie_metadata(data.first, data.second, aie_meta);
+  return ::get_event_tiles(aie_meta, graph_name, type);
```

Comment:
Are these top level functions could conflict with user-defined functions?

Reply:
@keryell I think impossible since ::get_event_tiles is in TU with internal linkage, and user-defined function is definitely not.